timer.hpp

```diff
@@ -198,29 +199,59 @@ AliquotTimer<F>& AliquotTimer<F>::operator()()
     double       now        = start_time;
     double       previous   = start_time;
     double       minimum    = limit;
+    // Initialize the variables to avoid `may be used uninitialized` error.
+    double       maximum    = 0.0;
+    double       last_m     = 0.0;
+    double       last_s     = 0.0;
+    // Compute the mean and the standard deviation on the fly using the
+    // algorithm from TAoCP Vol 2, section 4.2.2.
```

Comment:
```suggestion
    double       maximum    = 0.0;
    // These variables are used to compute the mean and the standard deviation 
    // on the fly using Welford's algorithm (see TAoCP Vol 2, section 4.2.2).
    double       last_m     = 0.0;
    double       last_s     = 0.0;
```

I don't think it's useful to say why we initialize them, when we already do it for the ones above and I'd like to keep `minimum` and `maximum` together. Also, let's use the proper name for this algorithm (which I didn't know when I had originally written this comment, which is now copy-and-pasted everywhere).